cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,4840 @@
+      "Epoch[9] Batch [199]\tSpeed: 100.998486 samples/sec\taccuracy=0.859062\tlr=0.010000\n",
+      "Epoch[9] Batch [249]\tSpeed: 101.056487 samples/sec\taccuracy=0.857750\tlr=0.010000\n",
+      "Epoch[9] Batch [299]\tSpeed: 101.008321 samples/sec\taccuracy=0.857292\tlr=0.010000\n",
+      "Epoch[9] Batch [349]\tSpeed: 101.136202 samples/sec\taccuracy=0.857143\tlr=0.010000\n",
+      "Epoch[9] Batch [399]\tSpeed: 101.136214 samples/sec\taccuracy=0.856563\tlr=0.010000\n",
+      "Epoch[9] Batch [449]\tSpeed: 101.048589 samples/sec\taccuracy=0.855833\tlr=0.010000\n",
+      "[Epoch 9] training: accuracy=0.855833\n",
+      "[Epoch 9] speed: 100 samples/sec\ttime cost: 71.524225\n",
+      "[Epoch 9] validation: top1=0.940000 top5=0.997500\n",
+      "Epoch[10] Batch [49]\tSpeed: 97.767507 samples/sec\taccuracy=0.880000\tlr=0.010000\n",
+      "Epoch[10] Batch [99]\tSpeed: 101.252844 samples/sec\taccuracy=0.865000\tlr=0.010000\n",
+      "Epoch[10] Batch [149]\tSpeed: 101.431820 samples/sec\taccuracy=0.855417\tlr=0.010000\n",
+      "Epoch[10] Batch [199]\tSpeed: 101.080777 samples/sec\taccuracy=0.858125\tlr=0.010000\n",
+      "Epoch[10] Batch [249]\tSpeed: 101.221539 samples/sec\taccuracy=0.853750\tlr=0.010000\n",
+      "Epoch[10] Batch [299]\tSpeed: 101.100850 samples/sec\taccuracy=0.848750\tlr=0.010000\n",
+      "Epoch[10] Batch [349]\tSpeed: 101.051407 samples/sec\taccuracy=0.850179\tlr=0.010000\n",
+      "Epoch[10] Batch [399]\tSpeed: 101.200691 samples/sec\taccuracy=0.850781\tlr=0.010000\n",
+      "Epoch[10] Batch [449]\tSpeed: 101.099799 samples/sec\taccuracy=0.849583\tlr=0.010000\n",
+      "[Epoch 10] training: accuracy=0.849583\n",
+      "[Epoch 10] speed: 100 samples/sec\ttime cost: 71.446160\n",
+      "[Epoch 10] validation: top1=0.932500 top5=0.995000\n",
+      "Epoch[11] Batch [49]\tSpeed: 97.903653 samples/sec\taccuracy=0.860000\tlr=0.010000\n",
+      "Epoch[11] Batch [99]\tSpeed: 101.107787 samples/sec\taccuracy=0.861250\tlr=0.010000\n",
+      "Epoch[11] Batch [149]\tSpeed: 101.331630 samples/sec\taccuracy=0.868750\tlr=0.010000\n",
+      "Epoch[11] Batch [199]\tSpeed: 101.022988 samples/sec\taccuracy=0.859375\tlr=0.010000\n",
+      "Epoch[11] Batch [249]\tSpeed: 101.064918 samples/sec\taccuracy=0.854250\tlr=0.010000\n",
+      "Epoch[11] Batch [299]\tSpeed: 100.944864 samples/sec\taccuracy=0.853958\tlr=0.010000\n",
+      "Epoch[11] Batch [349]\tSpeed: 101.133041 samples/sec\taccuracy=0.853036\tlr=0.010000\n",
+      "Epoch[11] Batch [399]\tSpeed: 101.067137 samples/sec\taccuracy=0.853125\tlr=0.010000\n",
+      "Epoch[11] Batch [449]\tSpeed: 101.035314 samples/sec\taccuracy=0.852639\tlr=0.010000\n",
+      "[Epoch 11] training: accuracy=0.852639\n",
+      "[Epoch 11] speed: 100 samples/sec\ttime cost: 71.493110\n",
+      "[Epoch 11] validation: top1=0.928750 top5=0.997500\n",
+      "[Epoch 12] EarlyStop after 10 epochs: no better than 0.94125\n",
+      "Applying 
```

Comment:
Going to make all plot comments here because the line numbers get a little wonky for figures:

- For the consensus labels vs ground truth + model labels vs ground truth plot, unsure if it's a github issue but the titles/figures and overlapping (hard to read). Also it might be nice (since the difference is not too significant) to make y-axis of both plots the same, just so we can clearly see that the crowdlab consensus labels outperform purely using model predicted labels

- [nit] For the Number of labels changed from prior itter + Number of unique indices added plot, I think it would be nice to make the x-axis ticks display whole numbers only, since 1.5 rounds does not make too much sense

- [nit] for all plots, "iterations" I think is a more commonly used term than "rounds" for x-axis label

- [nit] Capitalize "labels change" in the x-axis label (inconsistent with the rest)

Reply:
Follow up on "rounds" vs "iterations": since you are using rounds for clarity, I believe one of the plot titles should be "Number of labels changed from prior round" instead of itter 
(and just general reminder of checking consistency across notebook for rounds vs iter)